core/templates/dev/head/services/AudioPlayerService.js

```diff
@@ -33,8 +33,8 @@ oppia.factory('AudioPlayerService', [
       if (filename !== _currentTrackFilename) {
         AssetsBackendApiService.loadAudio(
         ExplorationContextService.getExplorationId(), filename)
-          .then(function(audioBlob) {
-            var blobUrl = URL.createObjectURL(audioBlob);
+          .then(function(loadedAudio) {
+            var blobUrl = URL.createObjectURL(loadedAudio.data);
```

Comment:
Is this an AudioFile object? If so call it loadedAudioFile

Reply:
Done